### PR TITLE
Add example notebook with irrigation experiment

### PR DESCRIPTION
**Pull request by Peter9192**
This PR adds an example notebook that shows how the eWaterCycle system can be used to quickly assess the impact of irrigation in an area.

### CODE REVIEW

**Comment by Peter9192** [comment]
@nielsdrost and @RolfHut , I have finished most of the work for this notebook. I want to run it again in one go, but since that takes a long time I was hoping you could already provide feedback on this version, so I can implement it already.

File: examples/Irrigation.ipynb [hunk 1]
```diff
@@ -0,0 +1,577 @@
+{
```
Comment (nielsdrost) [inline examples/Irrigation.ipynb:1]: <p>I think this needs a small comment on why this is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Review by nielsdrost — APPROVED** [review]
Awesome work @Peter9192! I'll leave it unmerged in case you have any further improvements you wanted to make.

**Comment by nielsdrost** [comment]
Merged, thanks @Peter9192 !

**Pull request merged**